site/docs/guides/create-derivation.md

```diff
@@ -0,0 +1,251 @@
+---
+sidebar_position: 2
+---
+# Create a derivation with flowctl
+
+Once you're familiar with creating a basic [Data Flow](../concepts/README.md#essential-concepts), you can take things a step further
+and transform your data with [derivations](../concepts/derivations.md).
+A **derivation** is a kind of Flow collection that results from the transformation of one or more other collections.
```

Comment:
I think this is a true statement, but I'm not entirely sure it's useful. I would probably frame this more as a derivation being a transformation applied to a collection, where the benefit is that your data is transformed in a consistent way, respecting the order/sequence in which it was captured. That transformation is then stored as a flow collection itself, which has the cute consequence of being able to write derivations _of other_ derivations (in addition to being able to materialize it to any of the supported destinations).

Reply:
@jgraettinger can you fact-check me here? Am saying things that are actually true? :P

Reply:
this is all true, but the docs context matters. this is pretty early on in the user's journey, and at this stage it's at the level of "oh derivations is how I munge my data".

I don't know how to pack what you said into a sentence that's as concise and approachable as what you're replacing.

Reply:
This is vaguely what I was thinking... while also totally understanding that writing isn't my job so absolutely feel free to ignore me completely @oliviamiannone 😅 

--- 

A derived collection takes data from an input such as a capture, and applies some transformation to that stream of data, keeping order intact. This transformed stream is itself stored as a flow collection, meaning that you can materialize it exactly the same way you would the original capture. 

::tip::
One consequence of this is that derivations can be based on other derivations, allowing you to build up complex and powerful flows of data, while also enabling the intermediary steps to be used as datasets in their own right.
```

Reply:
Thanks for this - I like where you're going with it and am going to riff on it a bit!